website/integrations/sources/discord/index.md

```diff
@@ -154,3 +154,205 @@ return user_matched
 ```
 
 Now bind this policy to the chosen enrollment and authentication flows for the Discord OAuth source.
+
+### Syncing discord roles to authentik groups
+
+:::info
+Ensure that the Discord OAuth source in 'Federation & Social login' has the additional `guilds.members.read` scopes added under the 'Protocol settings'.
+:::
+
+Create a new 'Expression Policy' with the content below, adjusting the variables where required:
```

Comment:
```suggestion
Create a new 'Expression Policy' with the content below, adjusting the variables where required.
```

Reply:
@Keyinator there was a colon here... did you intend to list out specific variables? Or was it supposed to be a period?